### PR TITLE
Remove UMASK from test data, deprecated

### PR DESCRIPTION
**Pull request by berland**
**Issue**
Deprecations warning popping out in test runs.

**Approach**
rm

### CODE REVIEW

**Review by jondequinor — APPROVED** [review]
Y/n
y

File: libres/old_tests/enkf/CMakeLists.txt [hunk 1]
```diff
@@ -78,7 +78,6 @@ foreach(
   gen_kw_logarithmic
   enkf_forward_init_GEN_KW
   enkf_forward_init_GEN_PARAM
-  enkf_umask_config
```
Comment (oyvindeide) [inline libres/old_tests/enkf/CMakeLists.txt:81]: Should delete the .cpp file also: libres/old_tests/enkf/test_enkf_umask_config.cpp

**Review by oyvindeide — CHANGES_REQUESTED** [review]

**Comment by berland** [comment]
Ouch, I think this is hit by #2553

**Review by oyvindeide — APPROVED** [review]

**Comment by oyvindeide** [comment]
Is it possible to fix? Or should we close this?

**Comment by berland** [comment]
Closed due to blockage.

**Pull request closed without merging**